controller/search.go

```diff
@@ -83,6 +83,12 @@ func (c *SearchController) Show(ctx *app.ShowSearchContext) error {
 
 	}
 	return application.Transactional(c.db, func(appl application.Application) error {
+		if ctx.Q == nil || *ctx.Q == "" {
+			jerrors, _ := jsonapi.ErrorToJSONAPIErrors(
```

Comment:
We could start logging the body for the requests of the bad parameter errors. But if the request implies a big json document, it can become really tedious to find things in the log traces. It isn't really common to log the whole body of a request in the logs of production envs.